### PR TITLE
Resume Form :] (Lawrence)

### PR DESCRIPTION
**Pull request by Christopher-Tom**
Lawrence was here.

### CODE REVIEW

File: api/resumeContents.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"resumeToAdd":{"personal":{"fullName":"","email":"","phoneNumber":"","github":""},"education":{"university":"","graduationDate":"","titleMajor":"","college":"","cumulativeGPA":"","courseWork":""},"experienceList":[{"organizationName":"","positionTitle":"","experienceLocation":"","experienceStartDate":"","experienceEndDate":"","experienceDescription1":"","experienceDescription2":"","experienceDescription3":""},{"organizationName":"","positionTitle":"","experienceLocation":"","experienceStartDate":"","experienceEndDate":"","experienceDescription1":"","experienceDescription2":"","experienceDescription3":""}],"projectList":[{"projectName":"","projectLocation":"","projectToolsUsed":"","projectStartDate":"","projectEndDate":"","projectDescription1":"","projectDescription2":"","projectDescription3":""},{"projectName":"","projectLocation":"","projectToolsUsed":"","projectStartDate":"","projectEndDate":"","projectDescription1":"","projectDescription2":"","projectDescription3":""}],"skills":{"proficient":"","experienced":"","familiar":""}}}
```
Comment (law-nguyen) [inline api/resumeContents.json:1]: add this file to gitignore

File: api/main_endpoints/routes/Resume.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+const router = express.Router();
+// const ResumeForm = require("../models/ResumeForm");
+
+
+const {
+  OK,
+  BAD_REQUEST,
+  UNAUTHORIZED,
+  FORBIDDEN,
+  NOT_FOUND,
```
Comment (law-nguyen) [inline api/main_endpoints/routes/Resume.js:11]: since these are unused, you can delete them

**Review by law-nguyen — CHANGES_REQUESTED** [review]
fix travis pls :>

also pls check my comments

**Review by IsitaB — COMMENTED** [review]
are we closing #625 ?

**Comment by law-nguyen** [comment]
I didn't realize that there was already a PR for this #625 , but there is new stuff here I believe so we'll close it. @Gremlinbaby  make sure you resolve the issues on both PRs in this one

File: api/main_endpoints/routes/Resume.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const router = express.Router();
+// const ResumeForm = require("../models/ResumeForm");
```
Comment (evanugarte) [inline api/main_endpoints/routes/Resume.js:3]:
```suggestion
```

File: src/Pages/Resume/Education.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {FormGroup, Input} from 'reactstrap';
+class Education extends React.Component{
+  render(){
+    if(this.props.currentStep !==1){
```
Comment (evanugarte) [inline src/Pages/Resume/Education.js:5]:
```suggestion
    if (this.props.currentStep !== 1) {
```

File: api/main_endpoints/routes/Resume.js (hunk 2 above)
Comment (Gremlinbaby) [reply api/main_endpoints/routes/Resume.js:11]: done

File: api/main_endpoints/routes/Resume.js (hunk 3 above)
Comment (Gremlinbaby) [reply api/main_endpoints/routes/Resume.js:3]: deleted line

File: src/Pages/Resume/Education.js (hunk 4 above)
Comment (Gremlinbaby) [reply src/Pages/Resume/Education.js:5]: added spaces

**Pull request closed without merging**